src/main/java/org/opensearch/knn/index/KNNMethod.java

```diff
@@ -81,6 +81,14 @@ public void validate(KNNMethodContext knnMethodContext) {
         methodComponent.validate(knnMethodContext.getMethodComponent());
     }
 
+    /**
+     * returns whether training is required or not
+     *
+     */
```

Comment:
Why is the return not specified in javadocs? I thought it had to be

Reply:
Ack